docs/guides/guards.md

```diff
@@ -106,6 +106,50 @@ console.log(searchAttempt3.actions);
 // => ['executeSearch']
 ```
 
+If you want to have a single event transition to different states in certain sitations you can supply an array of targets with conditions.
+
+For example you can model a door that listens for an `OPEN` event, and opens if you are an admin and error if you are not:
+
+```js
+var machine = new Machine({
+  id: 'door',
+  initial: 'closed',
+  states: {
+    closed: {
+      initial: 'idle',
+      states: {
+        'idle': {},
+        'error': {}
+      },
+      on: {
+        OPEN: [
+          { target: 'opened', cond: (extState) => extState.isAdmin },
+          { target: 'closed.error' }
+        ]
+      }
+    },
+    opened: {
+      on: {
+        CLOSE: 'closed',
+      }
+    },
+  }
+});
+
+var fullState = { isAdmin: true };
```

Comment:
Hmm, can you change these `var`s to `const` and give each state a unique name? Otherwise this looks great!

Reply:
```js
const firstState = ...
const secondState = ...
const thirdState = ...
```

Just to discourage mutation.

Reply:
Ah, I get you!

I'll fix it up this evening when I get a moment.

Reply:
Done!